server/templates/base.html.j2

```diff
@@ -11,39 +11,85 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <link rel="stylesheet" href="https://code.getmdl.io/1.3.0/material.indigo-blue.min.css">
     <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/style.css') }}">
-    <title>{% block title %}{{ exam.display_name }}{% endblock %}</title>
+    <title>{% block title %}{{ title }}{% endblock %}</title>
   </head>
 
   <body>
+    {# if there is a specific exam, store a variable to true#}
 
     <div class="mdl-layout mdl-js-layout mdl-layout--fixed-header">
       <header class="mdl-layout__header">
         <div class="mdl-layout__header-row">
-          <a href="{{ url_for('exam', exam=exam) }}">
-            <span class="hidden-small mdl-layout__title">
-              {{ exam.display_name }}
-            </span>
-          </a>
-          <div class="hidden-small mdl-layout-spacer"></div>
-          <nav class="mdl-navigation">
-            <a id="room-menu" class="mdl-navigation__link" href="#">
-              Rooms <i class="material-icons">expand_more</i>
+        
+          {% if exam %}
+            <a href="{{ url_for('exam', exam=exam) }}">
+              <span class="hidden-small mdl-layout__title">
+                {{ exam.offering.name + " - " + exam.display_name }}
+              </span>
+            </a>
+          {% elif offering %}
+            <a href="{{ url_for('offering', offering=offering) }}">
+              <span class="hidden-small mdl-layout__title">
+                {{ offering.name }}
+              </span>
             </a>
-            <ul class="mdl-menu mdl-menu--bottom-left mdl-js-menu" for="room-menu">
-              <a class="mdl-menu__item add-item" href="{{ url_for('new_room', exam=exam) }}"><i class="material-icons">add</i>
-                <span>Add rooms...</span>
+          {% else %}
+            <a href="{{ url_for('index') }}">
+              <span class="hidden-small mdl-layout__title">
+                {{ title }}
+              </span>
+            </a>
+          {% endif %}
+          
+          <div class="hidden-small mdl-layout-spacer"></div>
+
+          {% if exam %}
+            <nav class="mdl-navigation">
+              <a id="room-menu" class="mdl-navigation__link" href="#">
```

Comment:
This should be a `button` and needs an aria-expanded and aria-controls attributes

Reply:
Ok